src/sql/platform/angularEventing/browser/angularEventingServiceImpl.ts

```diff
@@ -24,8 +24,8 @@ export class AngularEventingService implements IAngularEventingService {
 		return emitter.event;
 	}
 
-	public sendAngularEvent(uri: string, event: AngularEventType, payload?: any): void {
-		const emitter = this._angularMap.get(uri);
+	public sendAngularEvent(uri: string | undefined, event: AngularEventType, payload?: any): void {
```

Comment:
was there a point in which we were passing in undefined here? I don't think undefined makes sense as an input here.

Reply:
Didn't really think if it makes sense or not, i'll change it to require string